pep-0594.rst

```diff
@@ -848,17 +528,6 @@ It also required a configure flag that caused an ABI incompatibility. The
 module was removed in 3.7 by Nathaniel J. Smith in
 `bpo-29137 <https://bugs.python.org/issue29137>`_.
 
-Module type
-  C extension + CAPI
-Deprecated in
-  3.7
-Removed in
-  3.7
-Has a designated expert
-   n/a
-Substitute
-  **none**
-
 
 Modules to keep
```

Comment:
Just for reference: grep.app hits for ``\n(import|from) {module}` for each of the modules:

* `colorsys`:
  993
* `fileinput`:
  1478
* `getopt`:
  4862
* `optparse`:
  12339
* `wave`:
  1569

And for their replacements:

* `argparse`:
  128400
* `colour`:
  769
* `colormath`:
  53
* `colorspacious`:
  7
* `pillow` (`import PIL`):
  39500

As an interesting twist, a couple of conversion functions in `colorsys` are actually used [one place](https://github.com/python-pillow/Pillow/blob/main/src/PIL/ImageColor.py) in Pillow (though they could easily be copied in, of course).